app/gitcoinbot/actions.py

```diff
@@ -20,66 +20,150 @@
+          "PS Make sure you're logged into Metamask!"
     return new_bounty_response
 
 
+def no_active_bounty(owner, repo, issue_id):
+    issue_link = f"https://github.com/{owner}/{repo}/issues/{issue_id}"
+    bounty_link = f"{settings.BASE_URL}funding/new?source={issue_link}"
+    no_active_bounty_response = f"No active bounty for this issue, consider create the bounty please [visit this link]({bounty_link}).\n\n " \
+                 "PS Make sure you're logged into Metamask!"
+
+    return no_active_bounty_response
+
+
 def parse_comment_amount(comment_text):
     re_amount = r'\d*\.?\d+'
     result = re.findall(re_amount, comment_text)
     return result[0]
 
 
+def parse_comment_currency(comment_text):
+    or_currencies = r'|'.join(CURRENCIES)
+    re_currencies = r'({})'.format(or_currencies)
+    result = re.findall(re_currencies, comment_text)
+
+    return result[0]
+
+
 def parse_tippee_username(comment_text):
     re_username = r'[@][a-z\d](?:[a-z\d]|-(?=[a-z\d])){0,38}'
     username = re.findall(re_username, comment_text)
     return username[-1]
 
 
+@Bound(amount_greater_than_zero)
 def new_tip_text(owner, repo, issue_id, comment_text):
     tip_amount = parse_comment_amount(comment_text)
+    tip_currency = parse_comment_currency(comment_text)
     username = parse_tippee_username(comment_text)
     issue_url = f'https://github.com/{owner}/{repo}/issues/{issue_id}'
-    tip_link = f'{settings.BASE_URL}tip/?amount={tip_amount}&username={username}&source={issue_url}'
+    tip_link = f'{settings.BASE_URL}tip/?amount={tip_amount}&tokenName={tip_currency}&username={username}&source={issue_url}'
     tip_response = f'To complete the tip, please [visit this link]({tip_link}).\n ' \
         'PS Make sure you\'re logged into Metamask!'
     return tip_response
 
 
-def claim_bounty_text(owner, repo, issue_id):
-    issue_url = f'https://github.com/{owner}/{repo}/issues/{issue_id}'
-    claim_link = f'{settings.BASE_URL}funding/details/?url={issue_url}'
-    claim_response = f'To finish claiming this bounty please [visit this link]({claim_link})'
-    return claim_response
+def start_work_text(owner, repo, issue_id):
+    start_work_link = f'{settings.BASE_URL}issue/{owner}/{repo}/{issue_id}'
+    start_work_response = f"To show this bounty as started please [visit this link]({start_work_link})"
+    return start_work_response
+
+
+def submit_work_text(owner, repo, issue_id):
+    submit_link = f'{settings.BASE_URL}issue/{owner}/{repo}/{issue_id}'
+    submit_response = f'To finish claiming this bounty please [visit this link]({submit_link})'
+    return submit_response
+
+
+def submit_work_or_new_bounty_text(owner, repo, issue_id):
+    issue_counter = Bounty.objects.filter(github_url=f'https://github.com/{owner}/{repo}/issues/{issue_id}').count()
+
+    if issue_counter > 0:
+        response_text = submit_work_text(owner, repo, issue_id)
+    else:
+        response_text = no_active_bounty(owner, repo, issue_id)
```

Comment:
This could probably changed to something like:

```python
def submit_work_or_new_bounty_text(owner, repo, issue_id):
    if Bounty.objects.filter(github_url=f'https://github.com/{owner}/{repo}/issues/{issue_id}').exists():
        return submit_work_text(owner, repo, issue_id)
    return no_active_bounty(owner, repo, issue_id)
```
or 
```python
def submit_work_or_new_bounty_text(owner, repo, issue_id):
    bounties = Bounty.objects.filter(github_url=f'https://github.com/{owner}/{repo}/issues/{issue_id}').exists()

    if bounties:
        response_text = submit_work_text(owner, repo, issue_id)
    else:
        response_text = no_active_bounty(owner, repo, issue_id)
    return response_text
```